packages/client/src/clients/feed/feed.ts

```diff
@@ -619,6 +619,10 @@ class Feed {
     // Emit the event that these items had their statuses changed
     // Note: we do this after the update to ensure that the server event actually completed
     this.broadcaster.emit(`items:${type}`, { items });
```

Comment:
Question: what is the difference between `this.broadcaster.emit()` vs `this.broadcastOverChannel()`? 

Asking because I'm wondering if we need to also `broadcastOverChannel()` with the `items.type` format? Currently it only seems to use the `:` format.

Reply:
I believe the `broadcastIOverChannel` has to do with cross-tab communication between feeds, where any event triggers a refetch of the current feed state: https://github.com/knocklabs/javascript/blob/05826b3115df6771f47874cb841b4dfc78f731cd/packages/client/src/clients/feed/feed.ts#L652

I'm not sure if we'd want to update this right now, since it all seems internal

Reply:
Ah okay, thanks for checking. If it's for internal usage, then yeah we can probably punt that for now and update when we do deprecate the other format.